src/pages/cplusplus/introduction-to-cplusplus/index.md

```diff
@@ -20,9 +22,11 @@ int main()
 ```
 
 ### The Output of this program will simply be :
-![Image of output](https://github.com/MrAsrani1312/Images/blob/master/snip1out.PNG)
+
+![Imgur](https://i.imgur.com/w84lDnT.png)
```

Comment:
Instead of using an image for this, could you make it a code snippet as well? That would save us from a lot of struggling with images.

```
Hello world!

Process returned 0 (0x0)    execution time : 0.016 s
Press any key to continue.
```